test/mocha/mock-data.js

```diff
@@ -3,9 +3,25 @@
  */
 'use strict';
 
+const {util: {clone}} = require('bedrock');
+
 const mocks = {};
 module.exports = mocks;
 
+const zcap = {
+  id: 'urn:zcap:076a866b-1a9b-49d9-989b-e86757ad7fd2',
+  allowedAction: ['test'],
+  controller: 'did:key:foo',
+  '@context': 'did:content:foo',
+  parentCapability: 'urn:zcap:bar',
+  invocationTarget: 'urn:test:055283db-2c39-466a-a1b8-b0c12007b25e',
+  referenceId: 'did:test:foo',
+  // TODO is this necessary?
```

Comment:
@aljones15 can you please provide additional detail here?  I believe you were explaining that some API request fails without the `invoker` being specified along with `controller` which seems redundant to me.

Reply:
Addressed here: https://github.com/digitalbazaar/bedrock-zcap-storage-http/pull/4/commits/f3ddda41204779d72aa3c168e697e7e2fdf4cc10

Reply:
@dlongley here's the code referenced in the TODO: https://github.com/digitalbazaar/bedrock-zcap-storage/blob/master/lib/storage.js#L240-L245

Does a ZCAP really need a controller AND an invoker?

Reply:
I've created an issuer for this as it does not need to hold up this PR

https://github.com/digitalbazaar/bedrock-zcap-storage-http/issues/5

Resolving this conversation.